appveyor.yml

```diff
@@ -1,7 +1,7 @@
 init:
   - git config --global core.autocrlf true
 build_script:
-  - build.cmd --quiet --parallel verify
```

Comment:
@ryanbrandenburg some of these come from the Universe template - https://github.com/aspnet/Universe/blob/dev/build-template-dotnet/appveyor.yml. If we find issues with these later, we'll bulk update all the repos. For now, I'd focus on code changes